Sources/StreamChat/Models/Channel.swift

```diff
@@ -274,7 +274,27 @@ extension ChatChannel: AnyChannel {}
 
 extension ChatChannel: Hashable {
     public static func == (lhs: ChatChannel, rhs: ChatChannel) -> Bool {
-        lhs.cid == rhs.cid
+        lhs.cid == rhs.cid &&
+            lhs.updatedAt == rhs.updatedAt &&
+            lhs.cooldownDuration == rhs.cooldownDuration &&
+            lhs.createdAt == rhs.createdAt &&
+            lhs.createdBy == rhs.createdBy &&
+            lhs.deletedAt == rhs.deletedAt &&
+            lhs.extraData == rhs.extraData &&
+            lhs.imageURL == rhs.imageURL &&
+            lhs.isFrozen == rhs.isFrozen &&
+            lhs.isHidden == rhs.isHidden &&
+            lhs.lastMessageAt == rhs.lastMessageAt &&
+            lhs.memberCount == rhs.memberCount &&
+            lhs.membership == rhs.membership &&
+            lhs.muteDetails == rhs.muteDetails &&
+            lhs.name == rhs.name &&
+            lhs.ownCapabilities == rhs.ownCapabilities &&
+            lhs.previewMessage == rhs.previewMessage &&
+            lhs.reads == rhs.reads &&
+            lhs.team == rhs.team &&
+            lhs.truncatedAt == rhs.truncatedAt &&
+            lhs.watcherCount == rhs.watcherCount
```

Comment:
Better to reuse the DiffKit implementation 🤔 Or we can copy the DiffKit implementation here, and then change the DiffKit's `isContentEqual` to just `lhs == rhs`

Reply:
this is better actually, since DiffKit is in UIKit and not needed in SwiftUI

Reply:
Copied and sorted but moved updatedAt to the top since it is the most probable one to change